src/wolfio.c

```diff
@@ -151,7 +151,21 @@ static int TranslateIoError(int err)
         return WOLFSSL_CBIO_ERR_CONN_CLOSE;
     }
 
+#if defined(_WIN32) && defined(DEBUG_WOLFSSL)
+	char errstr[1024] = { "\tGeneral error: "};
```

Comment:
Is there a define/enum you can use instead of hard coded 1024?
Some compilers do not like inline variable declaration. Ideally this needs to be in a {} section or at top of function. Same with `int offset`.

Reply:
> Is there a define/enum you can use instead of hard coded 1024?

You would think.  I looked through dozens of examples in github.com/microsoft/*   random other repos,  and the local MS sdks.    It is almost always hard coded.   There is FORMAT_MESSAGE_ALLOCATE_BUFFER  that is used often (where it allocs the buffer) but to do so we would have to then dynamically alloc a second buffer just to add the general error part to the message.  

Closest to consider might be `GAI_STRERROR_BUFFER_SIZE`
but I just defined our own WOLFSSL_STRERROR_BUFFER_SIZE

> Some compilers do not like inline variable declaration. Ideally this needs to be in a {} section or at top of function. Same with `int offset`.

Yeah my C code has become lazy over the years sorry about that:)